### PR TITLE
dd: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/osx/dd.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# dd
+
+> Convierte y copia un archivo.
+> Más información: <https://keith.github.io/xcode-man-pages/dd.1.html>.
+
+- Crea una unidad USB de arranque desde un archivo isohybrid (como `archlinux-xxx.iso`) y muestra el progreso:
+
+`dd if={ruta/al/archivo.iso}} of={{/dev/dispositivo_usb}} status=progress`
```
Comment (kbdharun) [inline pages.es/osx/dd.md:8]:
```suggestion
`dd if={{ruta/al/archivo.iso}} of={{/dev/dispositivo_usb}} status=progress`
```

File: pages.es/osx/dd.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Clona una unidad a otra unidad con un bloque de 4 MB, ignora el error y muestra el progreso:
+
+`dd if={/dev/dispositivo_de origen}} of={{/dev/dispositivo_de destino}} bs={{4m}} conv={{noerror}} status=progress`
```
Comment (kbdharun) [inline pages.es/osx/dd.md:12]:
```suggestion
`dd if={{/dev/dispositivo_de origen}} of={{/dev/dispositivo_de destino}} bs={{4m}} conv={{noerror}} status=progress`
```

File: pages.es/osx/dd.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Genera un fichero de 100 bytes aleatorios utilizando el controlador aleatorio del kernel:
+
+`dd if=/dev/urandom of={ruta/al/archivo_aleatorio}} bs={{100}} count={{1}}`
```
Comment (kbdharun) [inline pages.es/osx/dd.md:16]:
```suggestion
`dd if=/dev/urandom of={{ruta/al/archivo_aleatorio}} bs={{100}} count={{1}}`
```

File: pages.es/osx/dd.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Compara el rendimiento de escritura de un disco:
+
+`dd if=/dev/zero of={ruta/para/archivo_1GB}} bs={{1024}} count={{1000000}}`
```
Comment (kbdharun) [inline pages.es/osx/dd.md:20]:
```suggestion
`dd if=/dev/zero of={{ruta/para/archivo_1GB}} bs={{1024}} count={{1000000}}`
```

File: pages.es/osx/dd.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Genera una copia de seguridad del sistema en un archivo IMG y muestra el progreso:
+
+`dd if=/dev/{{dispositivo_unidad}} of={{ruta/al/archivo.img}} status=progress`
+
+- Restaura una unidad desde un archivo IMG y muestra el progreso:
+
+`dd if={ruta/al/archivo.img}} of={/dev/unidad_dispositivo}} status=progress`
```
Comment (kbdharun) [inline pages.es/osx/dd.md:28]:
```suggestion
`dd if={{ruta/al/archivo.img}} of={{/dev/unidad_dispositivo}} status=progress`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**